content/applications/productivity/iot/devices/screen.rst

```diff
@@ -0,0 +1,96 @@
+================
+Connect a Screen
+================
+
+When using your *IoT Box* in Odoo, you could need to use a screen.
+After being connected, it will allow you to display to your client the
+current PoS order.
+
+.. image:: screen/screen-pos-client-display.png
+   :align: center
+
+.. note::
+        You can also access the Customer Display from any other computer by
+        accessing the *IoT Box* homepage and clicking on the *POS Display* button.
+
+.. note::
+        Starting version 15. The usage of the *IoT Box* is no longer mandatory
+        to display the PoS session. You can set in the PoS configuration
+        to display the PoS session from any computer using the PoS.
```

Comment:
I agree with ANV that we should remove the mention of Odoo 15. 

Since this PR is targeting the `13.0` branch, it would be odd to mention Odoo 15. After the forward-ports are complete, we could open a new PR that targets the `15.0` branch to add this `note` about Odoo 15.